themes/src/main/resources/theme/keycloak.v2/account/src/app/PageHeaderTool.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import {PageHeaderTools} from '@patternfly/react-core';
+import {ReferrerLink} from './widgets/ReferrerLink';
+import {LogoutButton} from './widgets/Logout';
+
+declare const referrerName: string;
+
+export class PageHeaderTool extends React.Component {
```

Comment:
This could be a function component instead, class components are on the way out.

Reply:
I think that could be said of this entire theme :) We discussed refactoring all of these, but it was determined to be beyond the scope of the current task.